core/src/main/java/uk/ac/bbsrc/tgac/miso/core/service/printing/Driver.java

```diff
@@ -318,6 +306,12 @@ protected static void appendTruncated(int length, String str, Consumer<String> w
     }
   }
 
+  private static void appendZebraEscapedUnicode(StringBuilder b, String text) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this method into the anonymous class declared at line 249. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3398)